eloquentjs/2_1_looping_a_triangle.js

```diff
@@ -0,0 +1,61 @@
+/**Draws a right triangle utilizing the += operator*/
+function drawTriangle(height) { 
+    if (isNaN(height)) {
+        throw "Triangle height cannot be NaN";
+        return;
```

Comment:
No need to return after throw, throwing halts execution and bubbles an exception up the stack until something handles it or it crashes the program.